src/test/integration/web_interface/rest/test_rest_statistics.py

```diff
@@ -0,0 +1,36 @@
+from storage.db_interface_statistic import StatisticDbUpdater
+from test.integration.web_interface.rest.base import RestTestBase
+
+
+class TestRestStatistics(RestTestBase):
+
+    def setUp(self):
+        super().setUp()
+        self.stats_updater = StatisticDbUpdater(config=self.config)
+        self.stats_updater.update_statistic('file_type', {'file_types': [['application/gzip', 3454]],
+                                                          'firmware_container': [['application/zip', 3], ['firmware/foo', 1]]})
+        self.stats_updater.update_statistic('known_vulnerabilities', {'known_vulnerabilities': [['BackDoor_String', 1]]})
+
+    def tearDown(self):
+        self.stats_updater.shutdown()
+        super().tearDown()
+
+    def test_rest_request_all_statistics(self):
+        st = self.test_client.get('/rest/statistics', follow_redirects=True)
+
+        assert b'file_type' in st.data
```

Comment:
It would be good to also check some of the actual stats to make sure they were retrieved correctly (e.g. the 'firmware_container' stats)